data/strings/strings.txt

```diff
@@ -1846,6 +1846,48 @@
     zh-Hans = 书签
     zh-Hant = 書籤
 
+  [bookmarks_and_tracks]
+    comment = "Bookmarks and Tracks" dialog title
+    tags = android,ios
```

Comment:
Should iOS-strings changes also be committed in 'Regenerated' commit?

Reply:
yes